tools/apidiff/markdown/markdown.go

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package markdown
```

Comment:
The package `markdown` has been refactored to `tools/internal/markdown`, please use that module instead.

Reply:
fixed